interpolator/interpolator.F90

```diff
@@ -80,8 +80,6 @@
 !! </table>
 module interpolator_mod
 
-#include <fms_platform.h>
```

Comment:
It's difficult to tell from just a review that this is safe.  My inclination is that it is safe because of #374  but you make need to `use platform_mod` .  Did you make sure none of the other stuff in fms_platform.h` is being used?

Reply:
Yes, I check and interpolator doesn't use any of the other stuff in `fms_platform.h`.